docs/project-structure/genezio-configuration-file.md

```diff
@@ -114,6 +125,31 @@ Variables can be used in the scripts. Check the [Usage](#how-to-use-variables-in
 - `local`: `string` | `string[]` **Optional**
 
   A general purpose script that runs before starting the local testing environment.
+### Basic `backend` deployment
+```yaml
+# Learn more about Genezio YAML at https://genezio.com/docs/project-structure/genezio-configuration-file/
+# The name of the project.
+name: (•◡•)project-name(•◡•)
+# The region where the project is deployed.
+region: (•◡•)region(•◡•)
+# The version of the Genezio YAML configuration to parse.
+yamlVersion: 2
+backend:
+  # The root directory of the backend.
+  path: .
+  # Information about the backend's programming language.
+  language:
+    # The name of the programming language.
+    name: js
```

Comment:
```suggestion
    name: ts
```

Reply:
We prefer to incline towards `ts` as a best practice programming language because it's typed.